examples/interactive/cli-platform/src/lib.rs

```diff
@@ -78,11 +78,14 @@ pub extern "C" fn rust_main() -> i32 {
     let size = unsafe { roc_main_size() } as usize;
     let layout = Layout::array::<u8>(size).unwrap();
 
+    // TODO: can we be more efficient about reusing the String's memory for RocStr?
```

Comment:
In non-Rust languages, where we actually have a direct pointer to `argv`, we can use seamless slices in the future to do this. However, we have to be careful to make sure they're actually valid UTF-8 if we do this, since there's no guarantee that `argv` points to valid UTF-8 strings!

Reply:
Yeah, good call